Lib/pathlib.py

```diff
@@ -707,6 +665,62 @@ def match(self, path_pattern, *, case_sensitive=None):
                 return False
         return True
 
+
+class PurePath(_BasePurePath):
+    """Base class for manipulating paths without I/O.
+
+    PurePath represents a filesystem path and offers operations which
+    don't imply any actual filesystem I/O.  Depending on your system,
+    instantiating a PurePath will return either a PurePosixPath or a
+    PureWindowsPath object.  You can also instantiate either of these classes
+    directly, regardless of your system.
+
+    On Posix, calling ``bytes()`` on a path gives the raw filesystem path as
+    a bytes object, as encoded by ``os.fsencode()``. On Windows, the unicode
+    form is the canonical representation of fileysstem paths, and so calling
+    ``bytes()`` on a path is not recommended.
```

Comment:
Because it’s important in Python to understand the difference between referencing and calling a function, and the syntax is uniform so you can have `somefunc(args1)(args2)` or `dispatch[key](params)`, it’s a personal pet peeve to see parens used to denote functions like in old C documentation.  `bytes()` is not a function, `bytes` is!  But I know our sphinx doc do that (display `the bytes() function`) but at least the source code is `func:bytes` and there is a setting to add parens or not 🙂 

So take or leave this suggestion!

```suggestion
    a bytes object, as encoded by os.fsencode. On Windows, the unicode
    form is the canonical representation of fileysstem paths, and so calling
    bytes on a path is not recommended.
```

Reply:
Interesting, makes sense - I'll bear that in mind for future! But I'll resolve this thread as we're deleting the paragraph :)